src/filters/mux_isom.c

```diff
@@ -8464,7 +8464,7 @@ static const GF_FilterArgs MP4MuxArgs[] =
 			"- v1: use v1 signaling, ISOBMFF style (will mux raw PCM as ISOBMFF style)\n"
 			"- v1qt: use v1 signaling, QTFF style\n"
 			"- v2qt: use v2 signaling, QTFF style (lpcm entry type)"
-		, GF_PROP_UINT, "v0", "|v0|v0s|v1|v1qt|v2qt", 0},
+		, GF_PROP_UINT, "v0", "v0|v0s|v1|v1qt|v2qt", 0},
 	{ OFFS(ssix), "create `ssix` box when `sidx` box is present, level 1 mapping I-frames byte ranges, level 0xFF mapping the rest", GF_PROP_BOOL, "false", NULL, GF_FS_ARG_HINT_ADVANCED},
```

Comment:
the first pipe (`|`) is actually not an error

the values are meant to map to the `GF_AudioSampleEntryImportMode` enum whose first value (0) is for `GF_IMPORT_AUDIO_SAMPLE_ENTRY_NOT_SET` (https://github.com/gpac/gpac/blob/master/include/gpac/isomedia.h#L2637)

in order for the default value of the prop (`v0`) to map to the right constant (`GF_IMPORT_AUDIO_SAMPLE_ENTRY_v0_BS = 1`) it needs to be second on the list of possible values 

this is the source of at least some of the test fails of the PR

Reply:
Is it normal to have 6 (5 + 1) possible values for this arg. However there are only 5 values in `GF_AudioSampleEntryImportMode enum`?

Reply:
Is it normal to have 6 (5 + 1) possible values for this arg. However there are only 5 values in `GF_AudioSampleEntryImportMode` enum?

Reply:
hm, in general it's not great but less problematic to have extra values, they will just be ignored

however in this case the extra value is defined, but not it's not in the enum

see: https://github.com/gpac/gpac/commit/1e5ad5bb63bb26529bd04262693cd1f94f4b64e7#diff-e0502e3bb4133c2be5d6f040521b18af03378ede1e3dfb4c7c71910ef2a7136cR39 

I don't know why it's here instead of the enum (ping @jeanlf)  but it exists

Reply:
the 6th value (v2qt) is only internal to mp4mux and is defined here https://github.com/gpac/gpac/blob/master/src/filters/mux_isom.c#L40

So yes, it's normal.